test/mocks/services/mock_location_service.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+
+import 'package:geolocator/geolocator.dart';
+import 'package:mockito/mockito.dart';
+import 'package:niira/models/game.dart';
+import 'package:niira/models/location.dart';
+import 'package:niira/services/location_service.dart';
+
+class MockLocationService extends Mock implements LocationService {
```

Comment:
If you are overriding the methods of the class that suggests to me that the mock might be better as a fake or a real object. What do you think?

Reply:
i like it, just refreshed my brain with mockito docs and im super keen to change it to a fake :)